src/lexer.coffee

```diff
@@ -205,8 +209,11 @@ exports.Lexer = class Lexer
       [tagToken[2].first_line, tagToken[2].first_column] =
         [poppedToken[2].first_line, poppedToken[2].first_column]
     if colon
-      colonOffset = input.lastIndexOf ':'
-      @token ':', ':', colonOffset, colon.length
+      colonOffset = input.lastIndexOf if inCSXTag then '=' else ':'
+      colonToken = @token ':', ':', colonOffset, colon.length
+      colonToken.csxColon = yes if inCSXTag # used by rewriter
+    if inCSXTag and tag is 'IDENTIFIER' and prev[0] isnt ':'
```

Comment:
This is for attributes following bare attributes.
```jsx
<Component scrollable x="this would blow up" />
```
I'll see if I'm missing this test case.